seeds/primary.js

```diff
@@ -1,13 +1,48 @@
 
+/*
+return knex.schema.createTable('users', t => {
+			t.string('email').primary()
+			t.string('first_name')
+			t.string('last_name')
+			t.string('address')
+			t.string('pass_hash')
+			t.boolean('staff')
+		})
+*/
+
 exports.seed = function(knex) {
   // Deletes ALL existing entries
-  return knex('table_name').del()
+  return knex('users').del()
```

Comment:
Note that this will fail if other tables have data in them references something in this table.